packages/epo-react-lib/src/atomic/Button/Button.tsx

```diff
@@ -18,7 +17,7 @@ export interface ButtonProps extends HTMLProps<HTMLButtonElement> {
   isInactive?: boolean;
 }
 
-const Button = forwardRef<HTMLButtonElement, ButtonProps>(
+const Button = forwardRef<HTMLButtonElement, PropsWithChildren<ButtonProps>>(
```

Comment:
trying to switch to use `PropsWithChildren` interface instead of manually adding children to all props.

Reply:
Oh.....never have I ever.